napalm_junos/utils/junos_views.py

```diff
@@ -3,5 +3,11 @@
 """
 from jnpr.junos.factory import loadyaml
 from os.path import splitext
-_YAML_ = splitext(__file__)[0] + '.yml'
+from napalm_base.utils import py23_compat
+
+# Temporary fix until PYEZ provides a better solution
+if py23_compat.PY2:
```

Comment:
Please let's not have two separate files - not only that it's redundant, but also will be harder to maintain.

Till junos-eznc will provide the necessary features, we can do:

```
_YAML_ = splitext(__file__)[0] + '.yml'
_junos_views = loadyaml(_YAML_)
if py23_compat.PY3:
    for obj_name, obj in _junos_views.items():
        if not hasattr(obj, 'FIELDS'):
            # it's a table not a view
            continue
        for field_name, field_props in obj.FIELDS.items():
            if field_props.get('astype') == unicode:
                field_props['astype'] = str
globals().update(_junos_views)
```

Which goes through the views and replaces unicodes with text, _only_ when PY3.